modules/how-huge-pages-are-consumed-by-apps.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/what-huge-pages-do-and-how-they-are-consumed-by-apps.adoc
+
+[id='how-huge-pages-are-consumed-by-apps-{context}']
+= How huge pages are consumed by apps
+
+Nodes must pre-allocate huge pages in order for the node to report its huge page
+capacity. A node can only pre-allocate huge pages for a single size.
+
+Huge pages can be consumed through container-level resource requirements using the
+resource name `hugepages-<size>`, where size is the most compact binary
+notation using integer values supported on a particular node. For example, if a
+node supports 2048KiB page sizes, it exposes a schedulable resource
+`hugepages-2Mi`. Unlike CPU or memory, huge pages do not support over-commitment.
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: hugepages-volume-
+spec:
+  containers:
+  - securityContext:
+      privileged: true
+    image: rhel7:latest
+    command:
+    - sleep
+    - inf
+    name: example
+    volumeMounts:
+    - mountPath: /var/lib/hugepages
```

Comment:
```suggestion
    - mountPath: /dev/hugepages
```
The hugetlbfs on RHCOS is mounted on /dev/hugepages

Reply:
Thanks! Updated